qr_generator/lib/data/sqlite_persistence.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:sqflite/sqlite_api.dart';
+
+class SqlitePersistence {
+  static const databaseName = 'superformula.db';
+  static const seedTableName = 'seed';
+  Database db;
+
+  SqlitePersistence._(this.db);
+
+  static Future<SqlitePersistence> create() async =>
+      SqlitePersistence._(await database());
+
+  static Future<Database> database() async {
+    return openDatabase(
+      join(await getDatabasesPath(), databaseName),
+      onCreate: (db, version) {
+        return db.execute(
+          '''CREATE TABLE $seedTableName(
+            id TEXT PRIMARY KEY,
+            expiresAt STRING
+          )''',
+        );
+      },
+      version: 1,
+    );
+  }
+
+  // Getting list of  values
+  Future<List<Map<String, dynamic>>> getUniqueObjects() async {
+    final ret = await db.rawQuery('SELECT * FROM $seedTableName ');
+    return ret;
+  }
+
+// Getting list of  values by query
```

Comment:
**chore**: use `flutter format .` to fix this spacing issue plus many more across various files

Reply:
Done!